packager/VisibilityChecker.cc

```diff
@@ -323,7 +308,7 @@ class VisibilityCheckerPass final {
         auto loc = litSymbol.loc(ctx);
 
         auto otherFile = loc.file();
-        if (!otherFile.exists() || !otherFile.data(ctx).isPackaged()) {
+        if (!otherFile.exists()) {
```

Comment:
This was the only load bearing line. All the other lines were in service of this code (or better error messages for already-reported errors). As in, this is the only usage that fed into a decision to report or not report an error in the first place.